packages/meditrak-server/src/app/addRoutesToApp.js

```diff
@@ -101,6 +102,7 @@ export function addRoutesToApp(app) {
   app.get('(/v[0-9]+)/me/countries', getCountryAccessList);
   app.get('(/v[0-9]+)/answer/:recordId?', getAnswers);
   app.get('(/v[0-9]+)/disaster/:recordId?', getDisasters);
+  app.get('(/v[0-9]+)/surveyResponse/:recordId?', getSurveyResponses);
```

Comment:
I've also just realized that this should probably be `surveyResponses` as that's how the url the Admin Panel will send

Reply:
Fixed